firebase-storage/ktx/src/test/kotlin/com/google/firebase/storage/ktx/StorageTest.kt

```diff
@@ -105,6 +113,64 @@ class StorageTests : BaseTestCase() {
         assertThat(metadata.getContentType()).isEqualTo("text/html")
         assertThat(metadata.getCacheControl()).isEqualTo("no-cache")
     }
+
+    @Test
+    fun `ListResult destructuring declarations work`() {
+        val mockListResult = Mockito.mock(ListResult::class.java)
```

Comment:
You're mocking the final class `ListResult`, which ends up requiring the use of `mock-maker-inline` which according to [the docs](https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2) is still *incubating*. 

For this particular case, you can create a helper class in the package `com.google.firebase.storage` that can access  the package private constructor directly. We've already done something similar in 

https://github.com/firebase/firebase-android-sdk/blob/master/firebase-firestore/ktx/src/test/java/com/google/firebase/firestore/TestUtil.java

Reply:
@rlazo the package `com.google.firebase.storage` already had a `TestUtil` class, so I added a `listResult()` method in that same class. PTAL and let me know if that's ok

Reply:
👍